### ISSUE TITLE
New file_schema request

### ISSUE THREAD

**Issue by JohnOmernik** [dask/fastparquet#115]
In addition to simple and hive, could we have a "Drill" file scheme?  Drill could use the hive fileschema, but drill uses a dir(n) value rather then dir = key-value setup...

So instead of 4 partitions for 4 days in hive
mytable - 
      -day=2016-01-01
      -day=2016-01-02
      -day=2016-01-03
      -day=2016-01-04

Drill does this

mytable -
     -2016-01-01
     -2016-01-02
     -2016-01-03
     -2016-01-04

It allows for some easy reference with select * from mytable where dir0 = '2016-01-02' if it's nested then the next level is dir1 etc. 

So basically just another option for file_scheme that would output in the partitions without the key= part (and no metadata being written, just files in a partition scheme)

**Comment by martindurant** [comment]
For the first case of 'hive' scheme could you not do 
```
select * from mytable where dir0 = 'day=2016-01-02' 
```

By the way, apache-drill I'm sure is excellent and efficient in its queries (I have minimal experience), but have you considered querying using [dask.dataframe](http://dask.pydata.org/en/latest/dataframe.html), which has a read_parquet function (that uses fastparquet internally) and may align well with your pandas experience.

**Comment by JohnOmernik** [comment]
So you could do that, dir0 = 'day=2016-01-02' however, this is non-intuitive from a analyst stand point.  I can create a simple view in drill... CREATE view as select a, b, c, dir0 as `day` from mytable.   And then, without penalty, have the columns be intuitive. I can force the analyst to remember the scheme, or I can write some string munging around it (likely with a penalty) I guess, having the ability to define what whether or not the key is in the directory name would be handy.  

As to dsk vs. drill, it sounds like dask is more akin to Spark's dataframe. Drill is more of a BI tool that allows us to, without specifying a ton of details, query and join multiple data sets using just SQL. No programming required. I.e. ODBC, JDBC, etc interfaces only, no programming at all. I will look into Dask though.

**Comment by martindurant** [comment]
The reason I am hesitating (this is easy to implement), is that fastparquet will not be able to read in a directory structure created this way. Modifying the reader logic to cope would take more work - would it be correct to assume `dir*` as the field names? I would feel better if you could point me to some spec documentation in drill that describes this behaviour.
